cmd/evm/internal/t8ntool/execution.go

```diff
@@ -387,18 +398,23 @@ func MakePreState(db ethdb.Database, accounts types.GenesisAlloc) *state.StateDB
 		}
 	}
 	// Commit and re-open to start with a clean state.
-	root, err := statedb.Commit(0, false, false)
+	mptRoot, err := statedb.Commit(0, false, false)
```

Comment:
why change the var name here?

Reply:
to differentiate between the MPT root and a verkle root, this is not apparent here but for the transition there will be two concurrent roots. But I will revert for now, since there's not going to be any "rebase".